setup.py

```diff
@@ -28,6 +28,10 @@ def package_data(pkg, root_list):
     include_package_data=True,
     install_requires=[
         'XBlock',
+        'mitodl_xblock_utils==1.0.5',
```

Comment:
why `mitodl_xblock_utils`, it is working fine without it. is this part of this PR scope?

Reply:
Yes, the PR's scope is to add xblock-utils as a dependency, specifically our fork of it. It is a requirement for #185